docs/analyzer/user_guide.md

```diff
@@ -1541,6 +1542,42 @@ cross translation unit analysis arguments:
                         analysis. (default: parse-on-demand)
 ```
 
+### Taint analysis configuration <a name="taint"></a>
+
+Taint analysis is used to detect bugs and potential security-related errors
+caused by untrusted data sources.
+An untrusted data source is usually an IO operation in code, often related to
+the file-system, database, network, or environment variables.
+Taint analysis works by defining operations that introduce tainted values
+(`sources`), operations that cause taint to spread from tainted values
+(`propagators`), and operations that are sensitive to tainted values (`sinks`).
+Developers can also use an additional category of `filters` to express that some
+operations sanitize tainted values, and after sanitization,
+the value is trusted and safe to use.
+
+Taint analysis can be used with the default configuration by enabling the
+`alpha.security.taint.TaintPropagation` checker:
+```sh
+CodeChecker analyze -e alpha.security.taint.TaintPropagation
+```
+
+Taint analysis can be used with custom configuration by specifying the taint
+configuration file as a checker-option in addition to enabling the
+`alpha.security.taint.TaintPropagation` checker:
+```sh
+CodeChecer analyze -e alpha.security.taint.TaintPropagation --checker-config 'alpha.security.taint.TaintPropagation:Config=my-cutom-taint-config.yaml'
```

Comment:
This line is a little bit too long and I have to scroll horizontally if I want to see the end of this line:
![image](https://user-images.githubusercontent.com/6695818/143834869-f428da14-0911-4870-ab8d-5c1e5e37b180.png)

I think it is more readable this way:

```suggestion
CodeChecer analyze \
  -e alpha.security.taint.TaintPropagation \
  --checker-config 'alpha.security.taint.TaintPropagation:Config=my-cutom-taint-config.yaml'
```